Extensions/quick_tags.js

```diff
@@ -297,9 +302,6 @@ XKit.extensions.quick_tags = new Object({
 		$("#xkit-quick-tags-window").remove();
 		$("body").append(m_html);
 
-		$("#xkit-tag-input").bind("focus", XKit.extensions.quick_tags.cancel_menu_close);
-		$("#xkit-tag-input").bind("blur", XKit.extensions.quick_tags.menu_close);
```

Comment:
nevermind i get it now.

On Thu, Aug 6, 2015 at 1:26 PM WyohKnott <notifications@github.com> wrote:

> In Extensions/quick_tags.js
> <https://github.com/new-xkit/XKit/pull/150#discussion_r36448029>:
>
> > @@ -297,9 +302,6 @@ XKit.extensions.quick_tags = new Object({
> >  		$("#xkit-quick-tags-window").remove();
> >  		$("body").append(m_html);
> >
> > -		$("#xkit-tag-input").bind("focus", XKit.extensions.quick_tags.cancel_menu_close);
> > -		$("#xkit-tag-input").bind("blur", XKit.extensions.quick_tags.menu_close);
>
> That's the goal. The behaviour is now similar to one click postage, i.e.
> if one click outside of the popup, it closes, *unless* the focus was
> given to the input area, in which case it won't close (in order to do some
> copy-paste for example)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/150/files#r36448029>.
>